src/components/info-window.tsx

```diff
@@ -16,13 +16,15 @@ import {GoogleMapsContext} from './map';
 export type InfoWindowProps = google.maps.InfoWindowOptions & {
   onCloseClick?: () => void;
   anchor?: google.maps.Marker | google.maps.marker.AdvancedMarkerElement | null;
+  shouldFocus?: boolean | null;
```

Comment:
I think we can leave away the `| null` here. Optional and boolean should be enough.

```suggestion
  shouldFocus?: boolean;
```